classroom-template-bundle-spec.md

```diff
@@ -0,0 +1,168 @@
+default_locale          | ✓        | enum        | Must be a valid locale code
+version                 |          | string      | Version string used as content decorator
+classroom_type          |          | enum        | One of ["Self-paced", "Bootcamp/Workshop", "Instructor-led"]
+course_code             |          | string      | Id for Course, e.g. T_AHYXXX-I
+title                   | ✓        | dictionary  | A locale dictionary of titles
+description             | ✓        | dictionary  | A locale dictionary of descriptions
+objectives              |          | dictionary  | A locale dictionary of objectives
+audience                |          | dictionary  | A locale dictionary of audiences
+prerequisites           |          | dictionary  | A locale dictionary of prerequisites
+outline                 |          | dictionary  | A locale dictionary of outline elements usually formatted as a JSON string
+on_demand_url           |          | dictionary  | A locale dictionary of URL's to Coursera (or other) on-demand content
+tags                    |          | array       | Array of strings to be used as hints in searching, etc.
+product_tags            |          | array       | Array of strings from the "Products" column in [this sheet](https://docs.google.com/spreadsheets/d/1hUUch85HBRsRJsgRo9VCg0Pn7ZXi21sl6JU7VOr9LP8)
+role_tags               |          | array       | Array of strings from the "Roles" column in [this sheet](https://docs.google.com/spreadsheets/d/1hUUch85HBRsRJsgRo9VCg0Pn7ZXi21sl6JU7VOr9LP8)
+domain_tags             |          | array       | Array of strings from the "Domain" column in [this sheet](https://docs.google.com/spreadsheets/d/1hUUch85HBRsRJsgRo9VCg0Pn7ZXi21sl6JU7VOr9LP8)
+level                   |          | integer     | Integer between 1 and 4, with 1 being the easiest
+estimated_duration_days |          | integer     | Estimated time to take the course, in days
+estimated_duration      |          | integer     | Estimated time to take the course, in minutes
+lock_position           |          | boolean     | Allow lab order to be changed by the trainer
+max_hot_labs            |          | integer     | Maximum number of hot labs for this course.
+enable_drm              |          | boolean     | Enable DRM on instructions if supported
+resource_limit_check    |          | boolean     | Check classroom for fraud
+student_resources       |          | array       | Student-specific instructions. See the [Resource Spec](./resource-spec.md) for full specification.
+instructor_resources    |          | array       | Instructor-specific instructions. See the [Resource Spec](./resource-spec.md) for full specification.
+modules                 |          | array       | See below
+
+### Modules
+
+A `ClassroomTemplate` presents labs to the learner in a specific order. To keep some compatibilty with other "bundle entities" we will use a simple collection of steps, each with one lab activity.
```

Comment:
Will steps have one lab activity or have multiple lab activities like in the example below?

Reply:
Yep, multiple steps each with one lab activity ... I think that's what the example tries to represent.